dbml-homepage/docs/docs/README.md

```diff
@@ -419,6 +420,31 @@ If your enum values contain spaces or other special characters you can use doubl
         countries
     } 
 
+## Sticky Notes
+You can add sticky notes to the diagram canvas to serve as a quick reminder or to elaborate on a complex idea.
+
+    Note note_name {
+      'Note content'
+    }
+
+Example,
+
+    Table jobs {
+        id integer
+        status job_status
+    }
```

Comment:
Suggest removing these lines since I think 1 example is enough. Also, use '...' in table jobs to avoid users focusing on that code block.

Reply:
I simplified the example and moved the Sticky Notes block  between the Comments and Notes parts.